requirements.txt

```diff
@@ -1,9 +1,9 @@
--e git://github.com/tejado/pgoapi.git@1f25e907f3e5f1b603330e71041e1ad7bee7580f#egg=pgoapi
+-e git+https://github.com/tejado/pgoapi.git#egg=pgoapi
```

Comment:
I don't think so. @Grace we need to get `pgoapi` at that specific commit. Please don't remove the commit reference, otherwise ppl that install after/before others may get different versions of `pgoapi`.